filebeat/main.go

```diff
@@ -3,12 +3,9 @@ package main
 import (
 	"os"
 
-	"github.com/elastic/beats/filebeat/beater"
-	"github.com/elastic/beats/libbeat/beat"
+	"github.com/elastic/beats/filebeat/cmd"
```

Comment:
This change brings up the question if we should also change the generator to support this?